examples/plot_knockoff_aggregation.py

```diff
@@ -62,32 +68,46 @@ def single_run(
     )
 
     # Use model-X Knockoffs [1]
-    mx_selection = model_x_knockoff(X, y, fdr=fdr, n_jobs=n_jobs, seed=seed)
-
+    result = model_x_knockoff(
```

Comment:
IIUC, result is a tuple ? 
I don't like this: examples are here to expose the API, not to hide it.

Reply:
done